media/js/base/datalayer-page-id.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+if (typeof Mozilla == 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    var dataLayer = window.dataLayer = window.dataLayer || [];
```

Comment:
As we're now moving this out of the GTM snippet and into an external script, it would be worthwhile adding a comment here that states this is where the core `window.dataLayer` object is now defined?